.travis/install.sh

```diff
@@ -24,6 +24,10 @@ shopt -s extglob
 
 args=("$@")
 
+wget 'https://github.com/metomi/isodatetime/archive/master.tar.gz' \
```

Comment:
I have concerns about this - do we want to test what the end users sees if they run `pip install cylc-flow` in a clean environment (i.e. get the version of cylc-flow available on pypi) or do we test against master, which means that we are checking against the very latest version?

Reply:
I assumed you had this just temporarily, until `metomi-isodatetime` was released? Once that's done, then we update cylc-flow's `setup.py` replacing `isodatetime` by the new `metomi-isodatetime`.

Reply:
Yes. I suppose it can be. Perahaps I should add an @TODO comment here?

Reply:
Looking at the changes, we already have `setup.py` with the new `metomi-isodatetime`. And while this change here might fix Travis for now, you will have to revert anyway once `metomi-isodatetime` is released.

So I think it's best to remove this one, leaving `setup.py` with the new `metomi-isodatetime`, and the build broken in Travis. Once `metomi-isodatetime` is released, you trigger a new Travis build (if there's no way in their web interface, just close/open this PR) and then if the build passes and we have already reviewed, it is just a matter of re-approving & merging this PR.

WDYT?

Reply:
issue #3255 created

Reply:
Sorry I wasn't clear. I was suggesting to remove this line in `.travis.yml`, leaving Travis broken until `metomi-isodatetime` is released. Once it is released, we trigger a new build and review/merge it.

Even though this works on Travis, it will fail for developers, unless they manually install metomi-isodatetime.

Reply:
Sorry - Got it now!